.zuul.yaml

```diff
@@ -0,0 +1,22 @@
+- job:
+    name: dnf-run-tests
+    description: Run DNF tests
+    roles:
```

Comment:
roles tells Zuul to expose the ansible roles found in zuul-distro-jobs (https://pagure.io/zuul-distro-jobs/blob/master/f/roles) in that job ansible workspace (we use the repo-setup roles in that job run playbook)